Framework/Algorithms/inc/MantidAlgorithms/ReflectometryReductionOneAuto3.h

```diff
@@ -44,29 +44,22 @@ class DLLExport ReflectometryReductionOneAuto3
     std::string iVsLam;
   };
 
-  class RebinParams {
-  public:
+  struct RebinParams {
     RebinParams(const double qMin, const bool qMinIsDefault, const double qMax,
```

Comment:
You can simplify this further by removing the constructor - it isn't required if you use an initializer list when you create an instance (e.g. see [here](https://docs.microsoft.com/en-us/cpp/cpp/initializing-classes-and-structs-without-constructors-cpp?view=vs-2019)).